common/js/mastra/agent.ts

```diff
@@ -0,0 +1,45 @@
+import type { AgentRequest, AgentResponse, AgentContext } from "@agentuity/sdk";
+import { Agent, Memory } from '@mastra/core';
+import { openai } from '@ai-sdk/openai';
+
+export default async function Agent(
+  req: AgentRequest,
+  resp: AgentResponse,
+  ctx: AgentContext,
+) {
+  const memory = new Memory();
+  
+  const simpleWorkflow = {
+    start: () => ({
+      llm: {
+        messages: [
+          { role: 'system', content: 'You are a helpful assistant.' },
+          { role: 'user', content: req.input || 'Hello, who are you?' }
+        ],
+        model: openai('gpt-4o-mini'),
+      },
+      then: (response) => {
+        ctx.logger.info('Agent response:', response.content);
+        return { result: response.content };
+      }
+    })
+  };
+
+  const mastraAgent = new Agent({
```

Comment:
does it make sense to define the agent in global scope and then invoke the workflow when the Agent function is invoked per request?